pkg/endpointmanager/manager.go

```diff
@@ -192,10 +192,11 @@ func (mgr *endpointManager) InitMetrics() {
 		// would result in negative counts.
 		// It must be thread-safe.
 
-		metrics.Endpoint = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
-			Namespace: metrics.Namespace,
-			Name:      "endpoint",
-			Help:      "Number of endpoints managed by this agent",
+		metrics.Endpoint = metric.NewGaugeFunc(metric.GaugeOpts{
+			Namespace:        metrics.Namespace,
+			Name:             "endpoint",
+			Help:             "Number of endpoints managed by this agent",
+			EnabledByDefault: true,
```

Comment:
That is a good question. I made it like this because the original pattern was disabled by default unless specifically enabled. I can go either way.

Reply:
Agreed, but if the field is "Disabled" and you don't specify it, then the metric is enabled by default. Which is the opposite of what it was before. I personally like your suggestion better since we have more metrics enabled by default anyways. But perhaps someone from @cilium/metrics should way in

Reply:
In practice, most metrics should be enabled by default, so I would lean towards `Disabled`, as that would promote metrics being enabled by default. It's okay if some metrics are just `Disabled: true` by default, especially if they're for specific use-cases or have high cardinality.